.yamato/gym-interface-test.yml

```diff
@@ -1,5 +1,5 @@
 test_editors:
-  - version: 2019.4
```

Comment:
Did you intend to make all the version changes part of the PR?

Reply:
Yes. These tests were actually not running at all since the dependency (test-build) was not run for this version

Reply:
Looks like we need to run the standalone build in 2019.4 not 2019.3; then you can revert these.

Reply:
I tried that and it said the bokken image was not found.